determined-ai/determined
# fix: selector fixes in log viewer

## Description
[WEB-466]
[WEB-462]



## Test Plan
- Check if selector is visually updated 
- Check if selector works as expected (filtering)
- Check if state can get sync with filter inputs



## Commentary (optional)
Antd `Select` has the good-fit feature, so replaced `MultiSelect` with antd `Select`



## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[WEB-466]: https://determinedai.atlassian.net/browse/WEB-466?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WEB-462]: https://determinedai.atlassian.net/browse/WEB-462?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
